### DISCUSSION TITLE
How to get htmx to do request with target _blank

### DISCUSSION THREAD

**Discussion by jsantari** [bigskysoftware/htmx#1793] [Q&A] [answered]
Is there a way with htmx to make the request to target _blank?

**Comment by andryyy** [comment, marked as answer]
Wouldn’t it be best to use a normal `<a/>` here? The advantages of swapping do not apply. 

A way to archive this anyway could be using the beforeRequest event and call window.open(url, "_blank").

**Reply by jsantari** [reply]
That's the route I've gone. Was trying to setup dual purpose for some reports either in content of the page or to set a flag and generate them as PDF for printout.
